dsaps/cli.py

```diff
@@ -1,9 +1,14 @@
+import datetime
+import logging
 import time
 
 import click
+import structlog
 
 from dsaps import models
```

Comment:
I had to change this line to just `import models` for it to work properly. Not sure why...

Reply:
The reason you had to change this is because this package is not installed in your environment. You could do `pipenv install -e .` to do that, but that's not great because dsaps isn't really a dependency of dsaps. The best thing to do here is to take advantage of a neat pipenv feature and add the following to `Pipfile`:
```python
[scripts]
dsaps = "python -c \"from dsaps.cli import main; main()\""
```
Then you should be able to run `pipenv run dsaps` and everything is cool.